docs/docs/tutorials/creating-new-environments.md

```diff
@@ -1,181 +1,94 @@
+* `analyst` namespace - Users in the `analyst` group can view and `admin` group can view+edit the environments in this namespace
+* `developer` namespace - Users in the `developer` and `admin` groups can view+edit the environments in this namespace
+* `nebari-git` namespace - Everyone can view and `admin`s can edit
 
-6. Lastly, click on `Full Logs` to view the full output from the `conda` build.
+As an individual user, you also have a personal namespace with the same name as your Nebari username.
 
 :::note
-If you want to use your new environment in a Jupyter Notebook, don't forget to include `ipykernel` and `ipywidgets` in
-your environment’s `yaml` file, or it may not be visible in the list of available kernels.
+If you can "view" an environment, you can use it.
 :::
 
-## Step 3 - Installing packages via `conda` or `pip`
+## Select environments in editors
 
-To install new packages through either `conda` or `pip` you'll need to navigate back to the `Edit` page of your environment
-in the `conda-store` web interface.
+Instructions to select any environment you have access to in the following editing spaces:
 
-Adding `conda` packages to a `conda-store` environment via the command line, is not possible since the files are
-read-only in that context.
+* **JupyterLab** - In a Jupyter Notebook, click on the "Select Kernel" dropdown in the top-left corner, and select the environment.
 
-Additionally, adding `pip` packages via the command line is strongly discouraged. Not only do `conda` and `pip` not
-always play nice together, but it can lead to some inconsistencies across users' dependencies.
-This happens because when you `pip` install a package from the command line this is installed in your `.local` folder, so this changes will not be applied to other Nebari users.
+* **VS Code** - Click on the ⚙️ icon in the bottom-right to open `Settings` -> `Command Pallette`, and type "Python: Select Interpreter" and press <kbd>Enter</kbd> to get the list of environments to select from.
 
-:::note
-One exception to this rule is packages that you are actively developing.
-As you are building a package, you will likely want to install it as a _dev_ package.
-This can be done using:
-
-```bash
-pip install --no-build-isolation --user -e .
-```
+* **Terminal** - In the terminal window, you can use `conda` CLI commands like `conda activate <namespace>-<environment_name>` to activate the relevant environment and `conda env list` to view the list of available environments.
 
-Or, if you’re using `flit`, you can install with:
+## Special requirements
 
-```bash
-flit install -s
-```
+### Dask
 
-Please keep in mind that these are _NOT_ available to Dask workers!
-:::
+Include the [`nebari-dask` metapackage](https://anaconda.org/conda-forge/nebari-dask) in your environment to use Dask. This ensures you have the correct version of `dask-gateway` and the latest versions of `dask` and `distributed` libraries.
```

Comment:
Add note that the nebari-dask metapackage version should match nebari version

Reply:
also note again here that you have to be in the develop group to be able to access dask.